server/app/routes/product.$productId.tsx

```diff
@@ -49,36 +82,108 @@ export default function Route() {
         <h2 className="text-lg font-semibold">Recent Reports</h2>
       </div>
       <div className="spacing-y-4 mx-auto columns-sm">
-        {reports.map((report) => (
-          <Card key={report.id} className="mb-4 break-inside-avoid-column">
-            <Title className="font-semibold">{report.review.title}</Title>
-            {report.review.is_top_positive_review ? (
-              <p>
-                <b>Top Positive Review</b>
-              </p>
-            ) : null}
-            {report.review.is_top_critical_review ? (
-              <p>
-                <b>Top Critical Review</b>
-              </p>
-            ) : null}
-            <p>
-              <b>Date: </b>
-              {report.review.date_text}
-            </p>
-            <p>
-              <b>Verified Purchase: </b>
-              {report.review.verified_purchase ? "Yes" : "No"}
-            </p>
-            <p>
-              <b>"Found Helpful" Votes: </b>
-              {report.review.found_helpful_count}
-            </p>
-            <hr className="my-2" />
-            <p>{report.review.text}</p>
-          </Card>
-        ))}
+        {reports.map(
+          (report) =>
+            report.issues?.length > 0 && (
+              <Card
+                key={report.id}
+                className="mb-4 break-inside-avoid-column"
+                id={report.id}
+                decoration={selectedProduct === report.id ? "left" : ""}
+              >
+                <Title className="font-semibold">{report.review.title}</Title>
+                <div>
+                  <b>Issues</b>:{" "}
+                  {report.issues.map(
+                    (issue) =>
+                      issue.text && (
+                        <div key={issue.id} className="border-4">
```

Comment:
Stylistic suggestion
```suggestion
                        <div
                          key={issue.id}
                          className="my-3 rounded border-2 border-slate-100 bg-slate-50 p-2"
                        >
```